djautotask/api_rest.py

```diff
@@ -0,0 +1,359 @@
+import json
+import logging
+import re
+
+import requests
+from django.conf import settings
+from retrying import retry
+from django.core.cache import cache
+
+from djautotask.utils import DjautotaskSettings
+
+CW_RESPONSE_MAX_RECORDS = 500  # The greatest number of records Autotask
```

Comment:
This should be AT_RESPONSE_MAX_RECORDS. But is it being used anywhere? If not it can be removed.